docs/products/kafka/reference/kafka-metrics-prometheus.rst

```diff
@@ -1,7 +1,18 @@
 List of metrics available via Prometheus integration
 ==================================================================
 
-Below you can find a summary of every metric available via Prometheus for an Aiven for Apache Kafka service.
+The list of available metrics highly depends on the service usage, hence, there are a few metrics would be available to
```

Comment:
Why some metrics should be available or not? Does this depends on the components used?